### PR TITLE
feat: Use goravel console command

### PR DESCRIPTION
**Pull request by kkumar-gcc**
## 📑 Description
image
image
image
image

## ✅ Checks

- [x] My pull request adheres to the code style of this project
- [x] My code requires changes to the documentation
- [ ] I have updated the documentation as required
- [x] All the tests have passed

### CODE REVIEW

File: console/commands/new_command.go
Comment (hwbrzzl) [inline console/commands/new_command.go:1]: Can we add test cases for this file?

**Review by hwbrzzl — COMMENTED** [review]
Great work, I think the screenshot is out of date, please update it.

File: main.go [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+	"os"
+
+	frameworkconsole "github.com/goravel/framework/console"
+
+	"github.com/goravel/installer/console"
+	"github.com/goravel/installer/support"
+)
+
+func main() {
+	cliApp := frameworkconsole.NewApplication("Goravel Installer", support.Version, "go run . [global options] command [command options] [arguments...]", support.Version, false)
```
[the remark this answers is not in the document]
Comment (kkumar-gcc) [reply main.go:13]: No, I believe both have different use cases. Version is used when we call -v or --version flag. But usage is displayed in Name. So i think it's okay to have two different property.
image

image

File: cmd/main.go
[the remark this answers is not in the document]
Comment (kkumar-gcc) [reply cmd/main.go:1]: Yeah, I will clean this up.

File: console/commands/new_command.go [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+}
+
+// Extend The console command extend.
+func (receiver *NewCommand) Extend() command.Extend {
+	return command.Extend{
+		Flags: []command.Flag{
+			&command.BoolFlag{
+				Name:  "dev",
+				Usage: "Installs the latest 'development' release",
+			},
```
[the remark this answers is not in the document]
Comment (kkumar-gcc) [reply console/commands/new_command.go:40]: Okay

File: console/commands/new_command.go [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+		ctx.NewLine()
+		return nil
+	}
+	ctx.NewLine()
+	color.Infoln("App key generated successfully!")
+	ctx.NewLine()
+
+	color.Infoln("Application ready in [" + bold.Sprintf("%s", name) + "]. Build something amazing!")
+	ctx.NewLine()
+	color.Infoln("Are you new to Goravel? Please visit https://goravel.dev to get started. Build something amazing!")
```
[the remark this answers is not in the document]
Comment (kkumar-gcc) [reply console/commands/new_command.go:193]: Yeah, noticed it.

File: console/commands/new_command.go
Comment (kkumar-gcc) [reply console/commands/new_command.go:1]: Yeah, will add

File: console/commands/new_command.go [hunk 4]
```diff
@@ -0,0 +1,197 @@
...
+
+	// remove the directory if it already exists
+	if err := os.RemoveAll(path); err != nil {
+		color.Errorf("error while removing the directory : %s\n", err.Error())
+		return nil
+	}
+
+	// clone the repository
+	clone := exec.Command("git", "clone", "https://github.com/goravel/goravel.git", path)
+	err := ctx.Spinner("creating a \"goravel/goravel\" project at \""+name+"\"", console.SpinnerOption{
```
Comment (hwbrzzl) [inline console/commands/new_command.go:108]:
```suggestion
	err := ctx.Spinner("Creating a \"goravel/goravel\" project at \""+name+"\"", console.SpinnerOption{
```

File: console/commands/new_command.go [hunk 5]
```diff
@@ -0,0 +1,197 @@
...
+				color.Errorf("error while installing the dependecies : %s\n", err)
+				return
+			}
+		},
+	})
+	if err != nil {
+		color.Errorf("error while installing the dependecies : %s\n", err)
+		return nil
+	}
+	color.Successln("goravel installed successfully!")
```
Comment (hwbrzzl) [inline console/commands/new_command.go:158]:
```suggestion
	color.Successln("Goravel installed successfully!")
```

**Review by hwbrzzl — COMMENTED** [review]
Perfect 👍

**Review by hwbrzzl — APPROVED** [review]

**Pull request merged**